<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cam_num">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="currentText">
    <string>1</string>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Start Frame</string>
   </property>
  </widget>
  <widget class="QComboBox" name="startFrame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>End Frame</string>
   </property>
  </widget>
  <widget class="QComboBox" name="endFrame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5010</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/newPrefix/data/qrc/confirm.png</normaloff>:/newPrefix/data/qrc/confirm.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>760</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/newPrefix/data/qrc/play-button.png</normaloff>:/newPrefix/data/qrc/play-button.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/newPrefix/data/qrc/pause.png</normaloff>:/newPrefix/data/qrc/pause.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="lefttop_widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>151</width>
     <height>341</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="lefttop_Layout"/>
   </widget>
  </widget>
  <widget class="QWidget" name="leftdown_widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="leftdown_Layout"/>
   </widget>
  </widget>
  <widget class="QWidget" name="right_widget" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>1001</width>
     <height>781</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>781</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="right_Layout">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="videoBox">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="videoBox_heatmap">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="videoBox_traj">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_7">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>991</width>
           <height>691</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="tab_Layout">
          <item row="0" column="0">
           <widget class="QWidget" name="subtab_widget" native="true">
            <widget class="QWidget" name="gridLayoutWidget_8">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>991</width>
               <height>721</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="subtab_Layout"/>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
        <widget class="QLabel" name="cam1_box">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>330</width>
           <height>160</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="cam3_box">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>370</y>
           <width>330</width>
           <height>160</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="cam2_box">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>330</width>
           <height>160</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="cam4_box">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>570</y>
           <width>330</width>
           <height>160</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="cam6_box">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>200</y>
           <width>330</width>
           <height>160</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="cam5_box">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>10</y>
           <width>330</width>
           <height>160</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="cam7_box">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>360</y>
           <width>330</width>
           <height>160</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="cam8_box">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>560</y>
           <width>330</width>
           <height>160</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="subtab2_widget" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>971</width>
           <height>731</height>
          </rect>
         </property>
         <widget class="QWidget" name="gridLayoutWidget_9">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>981</width>
            <height>731</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="subtab2_Layout"/>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="exit_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="min_Button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QWidget" name="info_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="window_widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="window_Layout"/>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="info_Layout"/>
  </widget>
  <widget class="QLineEdit" name="choose_id">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose ID</string>
   </property>
  </widget>
  <widget class="QWidget" name="main_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>191</width>
     <height>671</height>
    </rect>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>671</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="main_Layout"/>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
